.jenkins/libcxx_tests.Jenkinsfile

```diff
@@ -101,44 +25,27 @@ def ACClibcxxTest(String label, String compiler, String build_type) {
     }
 }
 
-def cleanup(){
-    node("nonSGX") {
-        timeout(GLOBAL_TIMEOUT_MINUTES) {
-            cleanWs()
-            checkout scm
-            oe.deleteRG([XENIAL_RG, BIONIC_RG])
```

Comment:
@oprinmarius Is it possible to do something malicious with this functionality?

I.e. could one do oe.delete(master_rg) or would it fail?

Reply:
@BRMcLaren Master RG has delete lock enabled . But we won't be using this function in any of the jobs from now on.